src/System.CommandLine/Binding/SpecificSymbolValueSource.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.CommandLine.Binding
+{
+    internal class SpecificSymbolValueSource : IValueSource
+    {
+        public SpecificSymbolValueSource(ISymbol symbol)
+        {
+            Symbol = symbol;
+        }
+
+        public ISymbol Symbol { get; }
+
+        public bool TryGetValue(IValueDescriptor valueDescriptor, BindingContext bindingContext, out object value)
+        {
+            var optionResult = bindingContext.ParseResult.FindResultFor(Symbol);
```

Comment:
I'm missing something here as I expected this to also handle command arguments.

Reply:
It does. The variable is misnamed but it's operating on symbols generally.